docs/06-patterns/navigation.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Navigation
+description: Navigation patterns are crucial for ensuring users can seamlessly move through an application.
+hide_title: true
+---
+
+# Navigation <Badge text='ready' color='green'></Badge>
+Navigation patterns are crucial for ensuring users can seamlessly move through an application. In Grafana, two key components play a vital role in navigation: the Megamenu and Breadcrumbs. Let's delve into how these components contribute to effective navigation patterns.
+
+## Megamenu Navigation
+
+**Overview:**  
+The Grafana megamenu is the primary navigation component for all of Grafana, including core, plugins, and other apps. It exists in three states - opened, closed, and “docked” open. When open, the menu acts as an overlay. When docked, the page content responds to a narrowed viewport. 
+
+**When to Use:**  
+The megamenu provides access to levels 1, 2, and 3 of the Grafana information architecture. Levels beyond this should be handled by tabs on a page (level 4) or inline elements using headings, steppers or other design elements (level 5 and up).
+
+**Usage Guidelines:**  
+- Except in rare cases, cloud applications should not be placed at level 1 of the megamenu. Instead, they should be grouped together with similar applications.
+- Ensure all megamenu entries have a corresponding page or place to go for consistent breadcrumb behavior.
+- Level 1 content must have an icon next to it for visual consistency; other levels do not utilize icons.
+- Utilize a landing page for all overview/category pages.
+- Ensure all megamenu entries are in sentence case.
+- Consider standardizing labels used across Grafana and using nouns over verbs for page titles.
+- Be cautious with page title length to avoid truncation.
+
+**Dos:**  
+- Utilize a landing page for all overview/category pages.
+- Ensure all megamenu entries are in sentence case.
+- Consider standardizing labels used across Grafana and using nouns over verbs for page titles.
+- Consider the number of pages you place in the megamenu.
+- Be careful with page title length to avoid truncation.
+- Strive for consistency within the Grafana ecosystem.
+
+**Don'ts:**  
+- Redirect within the megamenu to compensate for a lack of a landing page.
+- Use title case.
+- Include external links.
+- Nest tabs to indicate levels 5 and up.
+
+---
+
+## Breadcrumbs Component Documentation
+
+Grafana’s topnav breadcrumbs are a key part of the overall navigation experience. They are the very top most popular part of the 2022-2023 navigation redesign, and so it is critical to the user experience that they behave in a consistent and uniform manner.
+
+**When to Use:**  
+Breadcrumbs are automatically generated by the page hierarchy, so they are used on all pages.
+
+**Usage Guidelines:**  
+- Breadcrumbs should be used for information architecture hierarchy and not for history.
+- Utilize a separate page for anything reflected as a breadcrumb.
```

Comment:
As I said previously, I'd avoid repeating the rules. Could it be a good idea that "Usage Guidelines" comprises "Dos" and "Don'ts" sections? They are all focused on usage guidance 🤔

Reply:
Addressed this 👍